storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -0,0 +1,59 @@
+[id="persistent-storage-csi-migration"]
+= CSI Automatic Migration
+
+toc::[]
+
+Starting with {product-title} 4.8, automatic migration for the following in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers is available:
+
+* Amazon Web Services (AWS) Elastic Block Storage (EBS)
+
+* OpenStack Cinder
+
+:FeatureName: CSI Automatic Migration
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+With automatic migration enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers. By default, automatic migration is disabled.
+
+.Procedure
+
+You can enable and disable CSI automatic migration using feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
+
+.Automatic migration to all in-scope CSI drivers:
```

Comment:
Here, "in-scope" is unclear to me, so similar to my previous comment, maybe something like "supported by automatic migration"?

Reply:
Also, these titles should precede the yaml examples as an indicator of the example itself, as opposed to a topic header or task.